lib/core/src/lib/comments/comments.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="adf-comments-container">
-    <div id="comment-header" class="adf-comments-header" role="heading">
+    <div id="comment-header" class="adf-comments-header" aria-level="3" role="heading">
```

Comment:
In different PR I had discussion with Michał about that and seems like using h1/h2/h3 etc. elements is better than aria-level probably. Please read below links: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-level
"Opt for using the h1 thru h6 elements instead."

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/heading_role
"This defines the text in the div to be the main heading of the page, indicated by being level 1 via the aria-level attribute. Opt for using the h1 thru h6 element instead."

And also interesting explaining here, please read first answer:
https://webmasters.stackexchange.com/questions/137159/for-seo-is-role-heading-aria-level-1-the-same-as-h1

If you think it is better please change :)

Reply:
i will check this @AleksanderSklorz :)